livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -46,6 +47,7 @@
 
   protected static final Logger LOGGER = LoggerFactory.getLogger(BaseLivyInterprereter.class);
   private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+  private static String SessionNotFound_Pattern = "\"Session '\\d+' not found.\"";
```

Comment:
Camel case and underscore together ? Can you user all caps separated by underscore, like we are doing it in rest of the project for "static String" ?